src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -16,11 +16,7 @@
 
 package com.google.codeu.data;
 
-import com.google.appengine.api.datastore.DatastoreService;
-import com.google.appengine.api.datastore.DatastoreServiceFactory;
-import com.google.appengine.api.datastore.Entity;
-import com.google.appengine.api.datastore.PreparedQuery;
-import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.*;
```

Comment:
Let's import all classes instead of import "*". This is Google recommended style. Generally, IDE should help you add these imports, and you would not need to touch them.